ArduPlane/system.cpp

```diff
@@ -336,6 +336,9 @@ void Plane::set_mode(enum FlightMode mode)
     crash_state.is_crashed = false;
     crash_state.impact_detected = false;
 
+    // reset external attitude guidance
+    guided_state.guiding_roll = false;
```

Comment:
@magicrub  That would preclude future changes for pitch and yaw.

Chat on gitter?